File: query-feature.js
const fs = require('fs');
const { execSync } = require('child_process');

// Try to find a way to query the database
try {
  // Check if we have any database tools available
  const dbPath = './features.db';

  if (fs.existsSync(dbPath)) {
    console.log('Found features.db at:', dbPath);
    console.log('File size:', fs.statSync(dbPath).size, 'bytes');

    // Try to use child_process to run a database query if possible
    console.log('\nAttempting to query feature #242...');
    console.log('Please run one of these commands manually:');
    console.log('1. sqlite3 features.db "SELECT * FROM features WHERE id = 242"');
    console.log('2. python -c "import sqlite3; conn = sqlite3.connect(\'features.db\'); print(conn.execute(\'SELECT id, category, name, description FROM features WHERE id = 242\').fetchone())"');
  } else {
    console.log('features.db not found');
  }
} catch (error) {
  console.error('Error:', error.message);
}

File: query_feature.py
import sqlite3
import json

conn = sqlite3.connect(r'C:\Users\207ds\Desktop\Apps\mlodash-new\mlo-dash-new\features.db')
cursor = conn.cursor()
cursor.execute('SELECT id, category, name, description, steps FROM features WHERE id = 242')
result = cursor.fetchone()

if result:
    print(f"id: {result[0]}")
    print(f"category: {result[1]}")
    print(f"name: {result[2]}")
    print(f"description: {result[3]}")
    print(f"steps: {result[4]}")
else:
    print("No feature found with id=242")

conn.close()
